versions/draft-3/SPEC.md

```diff
@@ -1656,21 +1657,30 @@ workflow workflow_a {
 ```
 
 ### Importing Structs
-A `struct` can be defined and imported from another WDL and follows similar rules to other imported objects
+Any `struct` defined within an imported WDL will be added to a global namespace and will not be a part of the imported wdl's namespace. If two structs
+are named the same it will be necessary to resolve the conflicting names. To do this, one or more structs may be imported under an
+alias defined within the import statement.
 
-```wdl
-import structs.wdl as helpers
+For example, if your current WDL defines a struct named `Experiment` and the imported WDL also defines another struct named `Experiment` you can
+alias them as follows:
 
-task a {
-    helpers.MyStruct p
-    
-    command {
-        echo ${ p.name }
-    }
-}
+```wdl
+import http://example.com/example.wdl as ex alias Experiment as OtherExperiment
+```
 
+In order to resolve multiple structs, simply add additional alias statements.
+```wdl
+import http://example.com/another_exampl.wdl as ex2 alia Parent as Parent2 alias Child as Child2 alias GrandChild as GrandChild2
```

Comment:
I'd split this over three lines for clarity. Also noticed a typo on `alia` as I did this, eg
```
import http://example.com/another_exampl.wdl as ex2 
  alia Parent as Parent2 
  alias Child as Child2 
  alias GrandChild as GrandChild2
```